pycognaize/document/field/section_field.py

```diff
@@ -38,10 +39,23 @@ def __init__(self,
                          group_key=group_key, confidence=confidence,
                          group_name=group_name)
         self._field_id = field_id
+        self._value = value
+        self._ocr_value = ocr_value
+
+    @property
+    def start_tag(self):
```

Comment:
Why not start and end instead of start_tag end_tag?

Reply:
Because if specifies what the start is (Coordinates, value, tag or etc)

Reply:
@davidcognaize